.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 Sebastian Pipping <sebastian@pipping.org>
+# Licensed under the MIT license, see file COPYING for the full text
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    commit-message:
+      include: "scope"
+      prefix: "Actions"
+    directory: "/"
+    labels:
+      - "enhancement"
+    schedule:
+      interval: "weekly"
```

Comment:
I think monthly would be sufficient. I don't mind being one month behind and would rather prefer low noise.

Reply:
Okay, done